### ISSUE TITLE
access tokens: add user_id and possible more user information in access token

### ISSUE THREAD

**Issue by lnielsen** [inveniosoftware/invenio-oauth2server#144]
The current access token returned by Invenio as very minimal information:

```
access_token	"ZmluZCBvdXQgaXMgdGhlcmUgc3VwcG"
expires_in	3600
refresh_token	"9ydCBmb3IgcmVmcmVzaCB0b2tlbnM="
scope	        "write"
token_type	"Bearer"
```

As minimum I think we should add a user id so given two access tokens the remote app can identity that it is for the same Invenio user.

We could add two scopes: ``user:email`` and ``user:profile`` that allows a service to ask for email and profile information from a user.

**Comment by lnielsen** [comment]
Information can be added to the response [here](https://github.com/inveniosoftware/invenio-oauth2server/blob/master/invenio_oauth2server/views/server.py#L129-L132)

- Invenio-Accounts/UserProfiles should likely not define the scopes since that will add circular dependency on oauth2server.

**Comment by lnielsen** [comment]
Closed by #148
